sqlglot/expressions.py

```diff
@@ -1296,6 +1296,9 @@ class ColumnConstraintKind(Expression):
 class AutoIncrementColumnConstraint(ColumnConstraintKind):
     pass
 
+class PeriodForSystemTimeConstraint(ColumnConstraintKind):
+    ar_types = {"this": False, "start": True, "end": True}
```

Comment:
Typo on the `ar_types` - it needs to be `arg_types`.

I understand that we need two children expressions for this: one for `start` and one for `end`. If this is the case, let's do `arg_types = {"this": True, "expression": True}` and update the the parser method accordingly.